app/src/main/java/com/brandon/forzenbook/network/ForzenApiService.kt

```diff
@@ -8,12 +8,14 @@ interface ForzenApiService {
 
     @GET(LOGIN_ENDPOINT)
     suspend fun getCode(
```

Comment:
Maybe this should be something like "loginWithEmailCode"

Reply:
Will include in FA-89 since modules don't play nice with git.